ambari-infra/ambari-infra-assembly/pom.xml

```diff
@@ -26,12 +26,12 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>ambari-infra-assembly</artifactId>
   <name>Ambari Infra Assembly</name>
-  <url>http://maven.apache.org</url>
+  <url>https://maven.apache.org</url>
```

Comment:
looks like this URL does not make any sense. Probably we should remove it


Element | Description
-- | --
packaging | defines packaging type such as jar, war etc.
name | defines name of the maven project.
url | defines url of the project.
dependencies | defines dependencies for this project.
dependency | defines a dependency. It is used inside dependencies.
scope | defines scope for this maven project. It can be compile, provided, runtime, test and system.